### ISSUE TITLE
Midterm Peer Review

### ISSUE THREAD

**Issue by AaronZang** [woshibobo/ORIE-4741-Project#9]
Hi all,

Our group is also working on predicting housing price! It's nice to see how you approached the questions and what kind of analysis you had done! Really inspiring.

First of all, I think you guys did a good job on data collection and feature selection. The analysis on features are clear and reasonable. And I agree that you need to only consider those variables that have close relationship with the housing price to avoid overfitting.

As for the feature engineering, how did you deal with those nominal features? I think using dummy coding would increase the number of variables significantly. It would be better if you could talk a little bit more about that. And as for the model, I think you definitely should try other loss functions and regularizers to avoid overfitting, or probably you can try bootstrapping (bagging).

In all, you guys are definitely on the right truck, and good luck on your project for the rest of the semester.

Best,
Shibo
